acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/dsl/jsonrpc/Eth.java

```diff
@@ -37,4 +40,14 @@ public Condition getWorkExceptional(final String expectedMessage) {
   public Condition accountsExceptional(final String expectedMessage) {
     return new ExpectEthAccountsException(transactions.accounts(), expectedMessage);
   }
+
+  public Condition getTransactionReceipt(final Hash transactionHash) {
```

Comment:
Both these method should share consistent names with the objects they return and their function i.e. `getTransactionReceipt` does not get the transaction receipt (it checks there is one present).